lib/mj-librsvg.js

```diff
@@ -0,0 +1,26 @@
+var rsvg = require('librsvg').Rsvg;
+var Readable = require('stream').Readable;
+
+exports.svg2png = function(svg, result, scale, width, height, callback, check) {
+  var s = new Readable();
+  var svgRenderer = new rsvg();
```

Comment:
Thanks! Since I'm not too experienced with streams: does L11 catch only errors from the streaming process (which is nice if the string was corrupted somehow) or does it also catch errors that rsvg itself has produced?